drivers/sensor/ltrf216a/ltrf216a.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023 Tridonic
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_LTRF216A_H_
+#define ZEPHYR_DRIVERS_SENSOR_LTRF216A_H_
+
+#include <zephyr/drivers/i2c.h>
+
+struct ltrf216a_data {
+	uint8_t sample[3];
+};
+
+struct ltrf216a_config {
+	struct i2c_dt_spec i2c;
+};
```

Comment:
Don't need a separate header file for this, just move these definitions `drivers/sensor/ltrf216a/ltrf216a.c`

Reply:
applied in the latest force push.